iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -977,22 +977,22 @@ def create_survey(self, v2_index_page, row):
     def map_survey_description(self, row):
         v2_survey_description = []
 
-        v2_image = self.image_map.get(row['image_id'])
-        if v2_image:
-            v2_survey_description.append({'type': 'image', 'value': v2_image.id})
-
         v1_introduction = row['introduction']
         if v1_introduction:
             v2_survey_description.append({'type': 'paragraph', 'value': v1_introduction})
 
         v1_survey_description = json.loads(row['description'])
         for block in v1_survey_description:
-            if block['type'] == 'paragraph':
+            if block['type'] in ['heading', 'paragraph', 'list', 'numbered_list']:
                 v2_survey_description.append(block)
             elif block['type'] == 'image':
                 image = self.image_map.get(block['value'])
                 if image:
                     v2_survey_description.append({'type': 'image', 'value': image.id})
+            elif block['type'] == 'page':
+                page = self.v1_to_v2_page_map.get(block['value'])
```

Comment:
I don't think this works, as `v1_to_v2_page_map` is populated as we go through the pages. So if a survey links to a page that hasn't been processed yet, the lookup will fail even though it shouldn't.
Also see #640 (maybe you can fix this as part of this PR as well).